.eslintrc

```diff
@@ -1,3 +1,11 @@
 {
-  "extends": "standard"
+  "extends": "standard",
+  "parserOptions": {
+    "sourceType": "script"
+  },
+  "rules": {
+    "strict": [
+      "error"
+    ]
+  }
```

Comment:
Yes, I also want it. But I see that you use some editor integration — "Editor integration is easier that way." If your editor support eslintrc in the package.json, we definitely should get rid of yet another file and move configuration to package.json